ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/DeniedRequests.tsx

```diff
@@ -40,14 +46,76 @@ const searchOptions: SearchOption[] = [
     IMAGE_SEARCH_OPTION,
 ];
 
+const sortFields = [
+    'Request Name',
+    'Requester User Name',
+    'Created Time',
+    'Request Expiry Time',
+    'Image Registry Scope',
+];
+const defaultSortOption = {
+    field: sortFields[0],
+    direction: 'desc',
+} as const;
+
 function DeniedRequests() {
     const { searchFilter, setSearchFilter } = useURLSearch();
     const { page, perPage, setPage, setPerPage } = useURLPagination(20);
+    const { sortOption, getSortParams } = useURLSort({
+        sortFields,
+        defaultSortOption,
+        onSort: () => setPage(1),
+    });
+
+    const vulnerabilityExceptionsFn = useCallback(
+        () =>
+            fetchVulnerabilityExceptions(
+                {
+                    ...searchFilter,
+                    'Request Status': ['DENIED'],
+                },
+                sortOption,
+                page - 1,
+                perPage
+            ),
+        [searchFilter, sortOption, page, perPage]
+    );
+    const { data, loading, error } = useRestQuery(vulnerabilityExceptionsFn);
 
     function onFilterChange() {
         setPage(1);
     }
 
+    if (loading && !data) {
+        return (
+            <Bullseye>
+                <Spinner isSVG />
+            </Bullseye>
+        );
+    }
+
+    if (error) {
+        return (
+            <PageSection variant="light">
+                <TableErrorComponent
+                    error={error}
+                    message="An error occurred. Try refreshing again"
+                />
+            </PageSection>
+        );
+    }
+
+    if (!data) {
+        return (
+            <NotFoundMessage
+                title="404: We couldn't find that page"
+                message="Pending vulnerability exception requests could not be found."
+            />
+        );
+    }
+
+    // @TODO: Add a "No results" type of view when there is no data to show. Also, add them for the other tables.
```

Comment:
👍 We need to do this for tables throughout Workload CVEs.

Reply:
I'll make another ticket for this TODO